x-pack/plugins/fleet/server/services/epm/elasticsearch/template/install.ts

```diff
@@ -93,10 +101,16 @@ const installPreBuiltTemplates = async (paths: string[], esClient: Elasticsearch
 
     if (content.hasOwnProperty('template') || content.hasOwnProperty('composed_of')) {
       // Template is v2
-      return esClient.indices.putIndexTemplate(esClientParams, esClientRequestOptions);
+      return retryTransientErrors(
+        () => esClient.indices.putIndexTemplate(esClientParams, esClientRequestOptions),
+        { logger }
+      );
```

Comment:
@kpollich any feedback on this approach before I wrap the rest of our ES API calls?

Reply:
I think this looks good to me. Only minor usage question I have would be exposing a `maxRequests` parameter rather than hard coding it, but I think that's a change we could easily make if we actually have a use case for it later.